1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.spec.ts

```diff
@@ -1,5 +1,58 @@
+/* 
 
-describe('password validator', () => {
+The system under test here should meet the following requirements:
 
-})
+- Between 5 and 15 characters long
 
+- Contains at least one digit
+
+- Contains at least one upper case letter
+
+- Return an object containing a boolean result and an errors key that — when 
+provided with an invalid password — contains an error message or type for 
+all errors in occurrence. There can be multiple errors at a single time.
+
+*/
+import { passwordValidator } from "./index";
+
+describe("password validator", () => {
+  describe("length checks", () => {
+    it("does not validate passwords fewer than 5 characters", () => {
+      expect(passwordValidator("1huM")).toStrictEqual({
+        isValid: false,
```

Comment:
It's good practice to validate fields in your response object separately (rather than using strict-equal)